Sources/SWBCore/Specs/Tools/CCompiler.swift

```diff
@@ -660,6 +660,19 @@ public class ClangCompilerSpec : CompilerSpec, SpecIdentifierType, GCCCompatible
         let sparseSDKSearchPaths = GCCCompatibleCompilerSpecSupport.sparseSDKSearchPathArguments(producer.sparseSDKs, headerSearchPaths.headerSearchPaths, frameworkSearchPaths.frameworkSearchPaths)
         commandLine += sparseSDKSearchPaths.searchPathArguments(for: self, scope: scope)
 
+        if let clangInfo = optionContext as? DiscoveredClangToolSpecInfo, !clangInfo.isAppleClang {
+            var filteredCommandLine: [String] = []
+            var iterator = commandLine.makeIterator()
+            while let arg = iterator.next() {
+                if arg == "-index-store-path" {
+                    _ = iterator.next()
+                } else {
+                    filteredCommandLine.append(arg)
+                }
+            }
+            commandLine = filteredCommandLine
+        }
+
```

Comment:
Should we just not add the flag in the first place based on vendor? Or is the idea that it would often come in via `OTHER_*FLAGS`?

Reply:
I agree that would be better. Should be able to do that by providing a lookup closure to `commandLineFromOptions` (near line 622) and overriding COMPILER_INDEX_STORE_ENABLE to NO.

Reply:
Sounds good. Working on it 😄